configuration/reports/reportdescriptors/dataexports/dispensingData.yml

```diff
@@ -10,23 +10,16 @@ parameters:
     type: "java.util.Date"
     label: "reporting.parameter.endDate"
 datasets:
-  - {key: "dispensing", type: "dispensingDataSetManager"}
+  - key: "dispensingdataexport"
+    type: "sql"
+    config: "sql/dispensingData.sql"
 designs:
   - type: "csv"
     properties:
       "filenameBase": "dispensingdataexport.{{ formatDate request.reportDefinition.parameterMappings.startDate \"yyyyMMdd\" }}.{{ formatDate request.reportDefinition.parameterMappings.endDate \"yyyyMMdd\" }}.{{ formatDate request.evaluateStartDatetime \"yyyyMMdd\" }}.{{ formatDate request.evaluateStartDatetime \"HHmm\" }}"
       "characterEncoding": "ISO-8859-1"
       "blacklistRegex": "[^\\p{InBasicLatin}\\p{L}]"
       "dateFormat": "dd-MMM-yyyy HH:mm:ss"
-    processors:
-      - name: "saveToDisk"
-        type: "disk"
-        runOnSuccess: "true"
-        runOnError: "false"
-        processorMode: "AUTOMATIC"
-        configuration:
-          saveLocation: "{{application_data_directory}}reports"
-          compressOutput: "true"
```

Comment:
I assume you are removing this since the report runs much faster now and it is no longer run automatically and saved overnight?